### PR TITLE
Add vibration extension for mobile devices

### PR DESCRIPTION
**Pull request by Wend1go**
Functions:
 - Single vibration
 - Vibration pattern
 - Stop vibration

(Need to create a Help page)

### CODE REVIEW

**Comment by blurymind** [comment]
This is flipping awesome! Thank you!  😸

File: Extensions/DeviceVibration/devicevibrationtools.js [hunk 1]
```diff
@@ -11,8 +11,9 @@ gdjs.deviceVibration = {};
  * Vibrate the mobile device.
  * @param {number} duration Value in milliseconds.
  */
-gdjs.deviceVibration.vibrationStart = function(duration) {
-  navigator.vibrate([duration]);
+gdjs.deviceVibration.startVibration = function(duration) {
+  if (typeof navigator == "undefined" || !navigator.vibrate) return
+    navigator.vibrate([duration]);
```
Comment (4ian) [inline Extensions/DeviceVibration/devicevibrationtools.js]: The way this is indented looks like `navigator.vibrate([duration])` is called if expression inside the if is true, while it's not the case. Could you correct the indentation and add an extra blank line between the if and the navigator.vibrate? Thanks?

**Comment by 4ian** [comment]
(Semaphoreci build is failing because yarn/npm registry seems to be down, we can ignore this error for now)

**Comment by 4ian** [comment]
🚀Let's merge this 🚀
Thanks again for doing this! Will be very useful. You can create the page and add it to the left menu of the wiki (http://wiki.compilgames.net/doku.php/gdevelop5/topnav) 😊

**Pull request merged**